src/components/ProofImageInputRow.vue

```diff
@@ -50,11 +50,25 @@
   </v-row>
 
   <v-row v-if="showProofImagePreviewList" class="mt-0">
-    <v-col v-for="proofImagePreview in proofImagePreviewList" :key="proofImagePreview" cols="6">
-      <v-card>
-        <v-card-text class="pa-2">
+    <v-col v-for="(proofImagePreview, index) in proofImagePreviewList" :key="proofImagePreview" cols="6">
+      <v-card class="d-flex flex-column" height="100%">
+        <v-card-text class="flex-grow-1 pa-2">
           <v-img :src="proofImagePreview" style="max-height:200px" />
         </v-card-text>
+        <v-divider />
+        <v-card-actions>
```

Comment:
will probably add the picture date as a label next to the delete button :thinking: